src/formio.wizard.js

```diff
@@ -250,7 +251,7 @@ export default class FormioWizard extends FormioForm {
       return this.options.buttonSettings.showCancel;
     }
     if (name === 'submit') {
-      return (nextPage === null) || (this.page === (this.pages.length - 1));
+      return (nextPage === null) || (this.page === (this.pages.length - 1)) && this.options.buttonSettings.showSubmit;
```

Comment:
question: could `options` or `buttonSettings` be null?

Reply:
I think the `defaults` function is in charge of that (line 236)